tests/dualtor/test_orchagent_active_tor_downstream.py

```diff
@@ -55,26 +55,26 @@ def stop_garp(ptfhost):
     ptf_t1_intf = random.choice(get_t1_ptf_ports(tor, tbinfo))
     logging.info("send traffic to server %s from ptf t1 interface %s", server_ipv4, ptf_t1_intf)
     server_traffic_monitor = ServerTrafficMonitor(
-        tor, vmhost, test_params["selected_port"],
-        conn_graph_facts, exp_pkt, existing=True
+        tor, ptfhost, vmhost, tbinfo, test_params["selected_port"],
```

Comment:
So have you tested if this test works on both single and dualtor testbeds?

Reply:
Yes, the test results above are from a single dualtor testbed(7050-2).